Cubical/Algebra/AbGroup/Base.agda

```diff
@@ -139,7 +139,7 @@ module _ ((G , abgroupstr _ _ _ GisGroup) : AbGroup ℓ) where
   AbGroup→CommMonoid .snd .CommMonoidStr.isCommMonoid .IsCommMonoid.·Comm = IsAbGroup.+Comm GisGroup
 
 isSetAbGroup : (A : AbGroup ℓ) → isSet ⟨ A ⟩
-isSetAbGroup A = isSetGroup (AbGroup→Group A)
+isSetAbGroup A = is-set (str A)
```

Comment:
This is the single case where the `open ...Str ...` method was impractical, since the file already opens multiple records with `is-set` fields.